gems/smithy/lib/smithy/views/client/shapes.rb

```diff
@@ -117,17 +117,17 @@ def filter_traits(traits)
           traits.except(*OMITTED_TRAITS)
         end
 
-        def shape_type_from_type(type)
+        def shape_name_from_type(type)
           msg = "Unsupported shape type: `#{type}'"
           raise ArgumentError, msg unless SHAPE_TYPES_MAP.include?(type)
 
           SHAPE_TYPES_MAP[type]
         end
 
-        def shape_type_from_id(id)
+        def shape_name_from_id(id)
           return PRELUDE_SHAPES_MAP[id] if PRELUDE_SHAPES_MAP.key?(id)
 
-          Model::Shape.relative_id(id)
+          Model::Shape.name(id).camelize
```

Comment:
This fixes a bug where constants in add_member are not camelized and do not exist.